### PR TITLE
Add existsexpiredtickets to rpcserver

### PR DESCRIPTION
**Pull request by alexlyp**

### CODE REVIEW

**Comment by cjepson** [comment]
The minor version for the JSON RPC server should be bumped, as you're introducing a new feature and the remaining the RPC server commands are stable. See http://semver.org/

**Comment by chappjc** [comment]
tACK

File: dcrjson/dcrdextcmds.go [hunk 1]
```diff
@@ -45,6 +45,19 @@ func NewExistsAddressesCmd(addresses []string) *ExistsAddressesCmd {
 	}
 }
 
+// ExistsExpiredTicketsCmd defines the existslivetickets JSON-RPC command.
```
Comment (cjepson) [inline dcrjson/dcrdextcmds.go]: fix comment

File: rpcserver.go [hunk 2]
```diff
@@ -1754,6 +1755,64 @@ func handleExistsAddresses(s *rpcServer, cmd interface{},
...
+	txHashBlob, err := hex.DecodeString(c.TxHashBlob)
+	if err != nil {
+		return nil, &dcrjson.RPCError{
+			Code: dcrjson.ErrRPCDecodeHexString,
+			Message: fmt.Sprintf("bad ticket hash blob (unparseable): %v",
+				err.Error()),
+		}
+	}
+
+	// It needs to be an exact number of hashes.
```
Comment (cjepson) [inline rpcserver.go]: Use the new function in dcrjson to decode a blob of hashes

**Pull request merged**